code/game/machinery/defib_mount.dm

```diff
@@ -0,0 +1,155 @@
+//Holds defibs and recharges them from the powernet
+//You can activate the mount with an empty hand to grab the paddles
+//Not being adjacent will cause the paddles to snap back
+/obj/machinery/defibrillator_mount
+	name = "defibrillator mount"
+	desc = "Holds and recharges defibrillators. You can grab the paddles if one is mounted."
+	icon = 'icons/obj/machines/defib_mount.dmi'
+	icon_state = "defibrillator_mount"
+	density = FALSE
+	use_power = IDLE_POWER_USE
+	idle_power_usage = 1
+	power_channel = EQUIP
+	req_one_access = list(access_medical, access_heads) //used to control clamps
+	var/obj/item/defibrillator/defib //this mount's defibrillator
+	var/clamps_locked = FALSE //if true, and a defib is loaded, it can't be removed without unlocking the clamps
+
+/obj/machinery/defibrillator_mount/New(location, direction, building = 0)
+	. = ..()
+
+	if(location)
+		loc = location
+
+	if(direction)
+		dir = direction
+
+	if(building)
+		pixel_x = (dir & 3)? 0 : (dir == 4 ? -30 : 30)
+		pixel_y = (dir & 3)? (dir == 1 ? -30 : 30) : 0
+
+
+/obj/machinery/defibrillator_mount/loaded/New() //loaded subtype for mapping use
+	. = ..()
+	defib = new/obj/item/defibrillator/loaded(src)
+	update_icon()
+
+/obj/machinery/defibrillator_mount/Destroy()
+	QDEL_NULL(defib)
+	. = ..()
```

Comment:
purely preference, though this is technically fine, but I always preferred to use `return ..()` if it's at the end; save `. = ..()` for when you're not using it at the end.